packages/discovery/src/discovery/config/DiscoveryOverridesBuilder.ts

```diff
@@ -0,0 +1,153 @@
+import { ContractParameters, DiscoveryOutput } from '@l2beat/discovery-types'
+import { assign, parse, stringify } from 'comment-json'
+import * as fs from 'fs/promises'
+
+import { ContractOverrides } from './DiscoveryOverrides'
+import {
+  MutableDiscoveryOverrides,
+  MutableOverride,
+} from './MutableDiscoveryOverrides'
+import { RawDiscoveryConfig } from './RawDiscoveryConfig'
+
+export class DiscoveryOverridesBuilder {
+  private readonly mutableOverrides: MutableDiscoveryOverrides
+
+  constructor(
+    private readonly output: DiscoveryOutput,
+    private readonly rawConfigWithComments: RawDiscoveryConfig,
+  ) {
+    this.mutableOverrides = new MutableDiscoveryOverrides(
+      this.rawConfigWithComments,
+    )
+  }
+
+  getContracts(): ContractParameters[] {
+    return [...this.output.contracts]
```

Comment:
Why do we need to spread it and later create a new array if the `.contracts` already is an array